src/fiber/channel.rs

```diff
@@ -4666,18 +4666,14 @@ impl ChannelActorState {
             let sent_tlc_value = self.get_offered_tlc_balance();
             debug_assert!(self.to_local_amount >= sent_tlc_value);
             if sent_tlc_value + tlc.amount > self.to_local_amount {
+                trace!(channel = ?self.get_id(), tlc_id = ?tlc.tlc_id, tlc_amount = tlc.amount, sent_tlc_value = sent_tlc_value, to_local_amount = self.to_local_amount, "Sending tlc exceeds local balance",);
```

Comment:
cleanup these trace!?

Reply:
Done in https://github.com/nervosnetwork/fiber/pull/487/commits/2874d9190fdeb9470219dd6011b5e68b13eb98dc